public/controllers/management/components/management/ruleset/decoder-info.js

```diff
@@ -26,32 +26,58 @@ import {
   cleanFileContent,
   cleanInfo,
   updateFilters,
-  cleanFilters
+  cleanFilters,
 } from '../../../../../redux/actions/rulesetActions';
 
+import { UI_ERROR_SEVERITIES } from '../../../../../react-services/error-orchestrator/types';
+import { UI_LOGGER_LEVELS } from '../../../../../../common/constants';
+import { getErrorOrchestrator } from '../../../../../react-services/common-services';
+const errorContext = 'WzDecoderInfo';
+
 class WzDecoderInfo extends Component {
   constructor(props) {
     super(props);
 
     this.rulesetHandler = new RulesetHandler(RulesetResources.DECODERS);
+
+    const handleFileClick = async () => {
+      try {
+        const result = await this.rulesetHandler.getFileContent(value);
+        const file = { name: value, content: result, path: item.relative_dirname };
+        this.props.updateFileContent(file);
+      } catch (error) {
+        const options = {
+          context: errorContext,
```

Comment:
**question**: Why the `context` property is a string equals to the component name?

**sugestion**: We could use the `.name` property of the class instead create a variable of type string with the class name, something like:
```
context: WzDecoderInfo.name,

**thought**: Should the `context` property be more descriptive? The property only has the value of the component name, but we could do some reference to the function/method that is failing. Maybe some convention as
```
COMPONENT_NAME.METHOD
```

so we could have a `context = "WzDecoderInfo.handleFileClick"`

Reply:
Thanks @Desvelao. We will implement this approach in the next version. First, we must test if the components have the correct name in the property. 
And about to add the method in the error handler context. I think we discussed it with the group but I don't remember why we not approved.